include/message_filters/subscriber.h

```diff
@@ -241,7 +309,7 @@ class Subscriber : public SubscriberBase, public SimpleFilter<M>
   rclcpp::Node * node_raw_ {nullptr};
 
   std::string topic_;
-  rmw_qos_profile_t qos_;
+  rclcpp::QoS qos_ {100};
```

Comment:
Maybe this should be a pointer rather than arbitrarily initialized to this.

`qos_` is only used in the `subscribe()` method which "resubscribes", but I wonder what should happen if that method is called after construction but before an explicit subscribe with arguments is made?

Some of the constructors call subscribe with arguments, so maybe this should be initialized in the constructor instead?

Either way, this arbitrary default doesn't make much sense to me.

Reply:
I have removed this and initialized it in constructors